File: interface/src/main/java/cn/itcast/core/service/SeckillService.java
package cn.itcast.core.service;


import cn.itcast.core.pojo.entity.PageResult;
import cn.itcast.core.pojo.good.Brand;
import cn.itcast.core.pojo.seckill.SeckillGoods;

import java.util.List;

public interface SeckillService {

    public PageResult findPage(SeckillGoods seckillGoods, Integer page, Integer rows);

    public SeckillGoods findOne(Long id);

    public List<SeckillGoods> findAll();

    public SeckillGoods findById(Long id);

    void add(SeckillGoods seckillGoods);

    void update(SeckillGoods seckillGoods);
}

File: web_shop/src/main/webapp/js/service/seckillService.js
//服务层
app.service('seckillService',function($http){
    this.findAll=function(){
        return $http.get('../seckill/findAll.do');
    }

    this.findPage=function(page,rows){
        return $http.get('../seckill/findPage.do?page='+page+'&rows='+rows);
    }
    this.search=function(page,rows,searchEntity){
        return $http.post('../seckill/search.do?page='+page+"&rows="+rows, searchEntity);
    }

    this.findById=function(id){
        return $http.get("../seckill/findById.do?id="+id);
    }

    this.save = function(entity){
        return $http.post("../seckill/save.do",entity);
    }

    this.update=function(entity){
        return $http.post("../seckill/update.do",entity);
    }
});